base0x644/docker/beaver.conf

```diff
@@ -0,0 +1,11 @@
+[beaver]
+redis_url: redis://monitoring:6379/0
+redis_namespace: logstash
+logstash_version: 1
+queue_timeout: 3600
+
+[/var/log/ebs-snapshot.log]
+format: json
+type: ebs-snapshot
+tags: ebs-snapshot, master
```

Comment:
why is the master tag in here? should this not be changed to include the stack tag as per the other log file configs for beaver